secator/runners/command.py

```
@@ -60,6 +60,9 @@ class Command(Runner):
 	cwd = None
 	cwd_isolated = False
 
+	# Extra environment variables for subprocess
+	extra_env = {}
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)